### PR TITLE
Made working function for inserting data from form into database.

### PR DESCRIPTION
**Pull request by FelixLetheren**

### CODE REVIEW

File: newFish.php [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <?php
-require_once 'php-files/functions.php'
+require_once 'php-files/functions.php';
+
+$db = databaseConnect();
+if(isset($_POST['name'])){
```
Comment (mallowmew) [inline newFish.php]: Is it going to be resilient enough if you don't check every required field?

**Review by mallowmew — COMMENTED** [review]
One question - otherwise looks good

File: newFish.php [hunk 2]
```diff
@@ -1,5 +1,17 @@
 <?php
-require_once 'php-files/functions.php'
+require_once 'php-files/functions.php';
+
+$db = databaseConnect();
+if (
+    isset($_POST['name']) &&
```
Comment (sjr-mayden) [inline newFish.php]: Operators at the start of the line please

File: php-files/functions.php [hunk 3]
```diff
@@ -60,3 +60,20 @@ function displayFish(array $fishFromDB):string
     }
     return $result;
 }
+
+/**
+ * @param $dataBase
+ * @param array $fishArray
+ * Takes an array and inputs the data into the database.
```
Comment (sjr-mayden) [inline php-files/functions.php]: Change inputs for inserts

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Pull request merged**